### PR TITLE
Changes Error Responses

### PR DESCRIPTION
**Pull request by isakim01**
Details on upcoming changes to error responses.

### CODE REVIEW

File: articles/tech-resources/Changes-to-API-HTTP-Responses-and-Error-Codes [hunk 1]
```diff
@@ -0,0 +1,3313 @@
+---
+title: Upcoming Changes to SparkPost API HTTP Responses and Error Codes
+description: Details on upcoming changes that will occur on January 12, 2019
```
Comment (ndurant-sp) [inline articles/tech-resources/Changes-to-API-HTTP-Responses-and-Error-Codes]: 1) Why was this whole file removed?
2) Need to reword this as possible change as we are "reevaluating" in May, does that mean we should remove the "Post Change" responses and just have this as an error code documentation? If we are re-evaluating in May, but not set in stone, should just not have any actual error code responses in it, but be a simple high level document of proposed changes?

Comment (isakim01) [reply articles/tech-resources/Changes-to-API-HTTP-Responses-and-Error-Codes]: 1.Was recommended to me to delete the original off of github and to create a new one.
2. okay

**Comment by isakim01** [comment]
Will leave off "We are currently re-evaluating whether or not to deprecate extended error codes from the SparkPost API HTTP Responses. In the meantime, extended error codes will remain in place, unaltered, until May 2019" as this is a support doc and we don't want to confuse customers. we can point them to the change log support article instead
